src/content/docs/browser/browser-monitoring/page-load-timing-resources/new-relic-cookies-used-browser.mdx

```diff
@@ -96,6 +96,10 @@ The default browser agent configuration does not retain any personal data as def
   </tbody>
 </table>
 
+## Local Storage [#local]
+
+The Browser Agent uses the sessionStorage API to set a timestamped identifier in the end-user's browser local storage and enable session tracking. The behavior is documented [here](/docs/browser/browser-monitoring/page-load-timing-resources/cookie-collection-session-tracking/). That page also includes instructions on how to disable session tracking for an app.
```

Comment:
```suggestion
The browser agent uses the sessionStorage API to set a timestamped identifier in the end user's browser local storage. This action enables session tracking, which is documented in [our doc about cookie collection and session tracking](/docs/browser/browser-monitoring/page-load-timing-resources/cookie-collection-session-tracking/). That page also includes instructions on how to disable session tracking for an app.
```

Reply:
@toobagrrl please confirm if this edit is good to go as is, or needs a subtle fix before we merge! thanks!